src/components/OptionsSelector.js

```diff
@@ -191,12 +192,18 @@ class OptionsSelector extends Component {
      * Completes the follow up actions after a row is selected
      *
      * @param {Object} option
+     * @param {Object} ref
      */
-    selectRow(option) {
+    selectRow(option, ref) {
+        this.textInput.setNativeProps({selection: {start: 0, end: this.props.value.length}});
         if (this.props.shouldFocusOnSelectRow) {
             this.textInput.focus();
         }
```

Comment:
This should be inside the `shouldFocusOnSelectRow` if block.

Reply:
> This should be inside the `shouldFocusOnSelectRow` if block.

I was thinking, shouldn't it be inside of input's `onFocus` method? 
It is essentially `selectTextOnFocus` for Android

Reply:
yeah. Good catch.

Reply:
I was wrong about this. It is not `selectTextOnFocus`, on Android keyboard is always focused, so if we call it inside `onFocus` it will never fire. So we need to call it inside the `selectRow` function
[Comment: ](https://github.com/Expensify/App/pull/7514#issuecomment-1028054696)
> Should text be selected after selecting a user on mobile platfroms too?

Reply:
I forgot the context of the first comment. That's why `This should be inside the shouldFocusOnSelectRow if block.`

Reply:
> I forgot the context of the first comment. That's why `This should be inside the shouldFocusOnSelectRow if block.`

Currently `shouldFocusOnSelectRow` is `false` on all of the pages where `OptionsSelector` is used. 
This should be called every time `selectRow` is called on native platforms, since we need to select text whenever input was focused, and on native platforms input is always focused.

Reply:
ok. I think we don't need this prop anymore. Moreover, we may need to put changes of this PR behind a flag. OptionSelector is a Common component and we may not need these changes everywhere. I will let you more about this later.